### ISSUE TITLE
TableData Parsing incorrect conversion of active_power field

### ISSUE THREAD

**Issue by jd-lara** [NREL-SIIP/PowerSystems.jl#567]
The data in the RTS systems is being parsed incorrectly into TableData, possibly an incorrect conversion of the base 

```
g = "223_STEAM_3"
get_active_power(get_component(ThermalStandard, sys, g)) = 5.768000000000002
(get_component(ThermalStandard, sys, g)).active_power = 1.4000000000000004
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 3.5
g = "221_CC_1"
get_active_power(get_component(ThermalStandard, sys, g)) = 7.0379999999999985
(get_component(ThermalStandard, sys, g)).active_power = 1.6999999999999997
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 3.55
g = "321_CC_1"
get_active_power(get_component(ThermalStandard, sys, g)) = 7.0379999999999985
(get_component(ThermalStandard, sys, g)).active_power = 1.6999999999999997
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 3.55
g = "121_NUCLEAR_1"
get_active_power(get_component(ThermalStandard, sys, g)) = 18.6516
(get_component(ThermalStandard, sys, g)).active_power = 3.96
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 4.0
g = "123_STEAM_3"
get_active_power(get_component(ThermalStandard, sys, g)) = 5.768000000000002
(get_component(ThermalStandard, sys, g)).active_power = 1.4000000000000004
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 3.5
g = "107_CC_1"
get_active_power(get_component(ThermalStandard, sys, g)) = 7.0379999999999985
(get_component(ThermalStandard, sys, g)).active_power = 1.6999999999999997
(get_active_power_limits(get_component(ThermalStandard, sys, g))).max = 3.55
```

cc. @odow

**Comment by claytonpbarrows** [comment]
```
julia> g = "223_STEAM_3"
"223_STEAM_3"

julia> get_active_power(get_component(ThermalStandard, sys, g))
3.5

julia> get_component(ThermalStandard, sys, g).active_power
0.8495145631067961

julia> 0.8495145631067961*4.12
3.5

julia> PowerSystems.get_base_power(get_component(ThermalStandard, sys, g))
412.0
```

**Comment by claytonpbarrows** [comment]
the components are being constructed with a `SYSTEM_BASE` setting rather than `DEVICE_BASE`.

**Comment by jd-lara** [comment]
closed via #568
